docs/docs/guides/migration_from_other_libs/ethers.md

```diff
@@ -0,0 +1,478 @@
+
+  However, it is not advised to use the privatekey directly. And you are advised to use a secret storage or a vault instead.
+:::
+
+### Get unlocked account
+
+With ethers.js:
+
+```typescript
+const signer = await provider.getSigner();
+```
+
+With web3.js:
+
+```typescript
+const account = (await web3.eth.getAccounts())[0];
+```
+
+
+## Sending Transactions
+
+Sending a transaction with ethers.js:
+
+```typescript
+async function sendTransaction() {
+  const signer = new ethers.Wallet(privateKey, provider);
+
+  const tx = await signer.sendTransaction({
+    to: '0x92d3267215Ec56542b985473E73C8417403B15ac',
+    value: ethers.parseUnits('0.001', 'ether'),
+  });
+  console.log(tx);
+}
+
+sendTransaction();
+```
+
+With web3.js:
+
+:::info
+The method `web3.eth.sendTransaction` will use the account that you pass the public address at `from` to sign the transaction.
+
+So, the `from` needs to be the public address of a private key that you added previously to the web3.eth.accounts. Or, else, it would pass it to the provider where an unlocked account would be used. 
+
+And for the case when you did not add the private key early, and so the `from` was just passed to the provider. Then if the provider was a browser-injected provider like metamask, for example, it will ask the user to sign. And, if you are using a local dev node as a provider, it should be one of the accounts that were already unlocked at the node. However, note that it is highly risky and not recommended to unlock an account at a production or even a test node.
+:::
+
+```typescript
+async function sendTransaction() {
+	const web3 = new Web3('http://localhost:8545');
+
+	// The method web3.eth.sendTransaction is helpful if you are using a browser-injected provider like metamask.
+	// Or, if you are using a local dev node like ganache; and you have some accounts already unlocked at the node.
+	// And this is how you would get the first unlocked account from a local node (not advised for production or even on test node to use unlock accounts on the node).
+	const account = (await web3.eth.getAccounts())[0];
+  
+	// Alternative to the above, here is how to add wallet to be used as a signer later:
+	const wallet = web3.eth.accounts.wallet.add(privateKey);
+	const account = wallet[0].address;
+
+	const tx = await web3.eth.sendTransaction({
+		from: account,
+		to: '0x92d3267215Ec56542b985473E73C8417403B15ac',
+		value: web3.utils.toWei('0.00000000001', 'ether'),
+	});
+	console.log(tx);
+}
+
+sendTransaction();
+```
+
+## Sending a Signed Transactions
+
+Posting a signed transaction to the node with ethers.js:
+
+```typescript
+// in v5
+provider.sendTransaction(signedTx)
+
+// in v6
+provider.broadcastTransaction(signedTx);
+```
+
+With web3.js:
+
+```typescript
+async function sendSignedTransaction() {
+  const transaction: Transaction = {
+    from: senderPublicAddress,
+    to: receiverPublicAddress,
+    value: 1,
+    gas: 21000,
+    type: 1,
```

Comment:
Any specific reason to using a type 1 transaction rather than a type 2 transaction? I believe most people will use either type 0 or type 2